cmd/info/info.go

```diff
@@ -0,0 +1,69 @@
+package info
+
+import (
+	"github.com/CircleCI-Public/circleci-cli/api/info"
+	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/olekukonko/tablewriter"
+
+	"github.com/spf13/cobra"
+)
+
+//infoOptions info command options
+type infoOptions struct {
+	cfg       *settings.Config
+	validator validator
+}
+
+type validator func(cmd *cobra.Command, args []string) error
```

Comment:
can this `type` be refactored/extracted? I see that the same type is also defined in `cmd/policy/policy.go` [here](https://github.com/CircleCI-Public/circleci-cli/blob/master/cmd/policy/policy.go#L25) and in `cmd/runner/runner.go` [here](https://github.com/CircleCI-Public/circleci-cli/blob/master/cmd/runner/runner.go#L52).